docs/installation/azuread.asciidoc

```diff
@@ -79,3 +80,48 @@ You can perform 802.1X authentication of users using Azure AD but this will only
 . Restart radiusd using `/usr/local/pf/bin/pfcmd service radiusd restart`
 . All the users matching this realm will now authenticate against Azure AD. Make sure you also have a connection profile with auto-registration enabled and the Azure AD source in it so that your users are correclty authorized when connecting.
 
+===== Using Azure AD EAP-TLS machine authentication
+
+You can perform a EAP-TLS authentication and verify the machine group membership in order to provide a access to the network.
```

Comment:
perform a EAP-TLS -> perform EAP-TLS
provide a access to the network -> provide network access